experiments/tf_trainer/keras_gru_attention/run.py

```diff
@@ -36,7 +36,7 @@
 # TODO: Missing fields are not handled properly yet.
 LABELS = {
     "frac_neg": tf.float32,
-    #"frac_very_neg": tf.float32
+    "frac_very_neg": tf.float32
```

Comment:
Is there a plan to make the labels a flag?

Reply:
Maybe. I think I will first experiment a bit with the idea we discussed where I associate the heads (and labels with them) to the dataset_input. My thought is that the labels are inherently linked to a dataset